DEVELOPMENT.md

```diff
@@ -198,6 +198,8 @@ Run `npm run storybook:build` to build the static storybook files. The build art
 
 ## API definitions
 
+API definitions are documented in the Open API format in addition to the document below.  Please maintain the swagger/tekton-dashboard-api.yaml as well as this file when the API difinitions are changed.  The swagger UI  is available as the extension of the tekton dashboard.  Refer to swagger/README.md to enable it. 
```

Comment:
- Typo here (`definitions`), and some dubious whitespace
- `Tekton dashboard` (casing)
- Let's use `code` markdown too when referencing files
- `OpenAPI` not `Open API`, and we should add a link to https://swagger.io/docs/specification/about/

Reply:
I fixed all